scripts/comicdb.js

```diff
@@ -66,4 +66,40 @@ jQuery(document).ready(function($) {
       $('.form-add-list').addClass('active');
     }
   }
+
+  // Sort display controls
+  var $sortControls, $comics, $comicsList;
+
+  $sortControls = $('.sort-control');
+  $comicsList = $('#inventory-table');
+  $comics = $($comicsList).find('li');
+
+  $($sortControls).click(function() {
+    var controlId, layoutThumbLg, layoutThumbSm, layoutList;
+    controlId = $(this).attr('id');
+    layoutThumbLg = 'col-xs-6 col-sm-4 col-md-3 col-lg-2';
+    layoutThumbSm = 'col-xs-4 col-sm-3 col-md-2 col-lg-1';
+    layoutList = 'col-xs-12';
+
```

Comment:
Above are our three layouts, `layoutThumbLg`, `layoutThumbSm`, and `layoutList`. The text contained inside the string are the classes to apply to the `li`'s inside the `#inventory-table`.